test-src/org/pentaho/reporting/platform/plugin/cache/FileSystemCacheBackendTest.java

```diff
@@ -135,7 +148,7 @@ public String getValue() {
 
     private void writeObject( final ObjectOutputStream oos ) throws IOException {
       try {
-        Thread.sleep( 3000L );
+        Thread.sleep( 1000L );
```

Comment:
@tmorgner it's not enforcing order, it's mocking of writing/reading real objects that usually take some time, otherwise file locks will be released immediately